scripts/monorepo/runPublished.js

```diff
@@ -29,16 +29,24 @@ const websitePackages = [
 // in the root package.json's publishing-related scripts and will need to be updated if --scope changes.
 const beachballPackageScopes = Object.entries(getAllPackageInfo())
   .filter(([, { packageJson, packagePath }]) => {
-    // Ignore northstar
-    if (/[\\/]fluentui[\\/]/.test(packagePath)) {
+    const isNorthstar = /[\\/]fluentui[\\/]/.test(packagePath);
+
+    if (isNorthstar) {
+      return false;
+    }
+
+    if (packageJson.private === true) {
       return false;
     }
```

Comment:
nit: I'm generally a fan of not splitting them up, that's just me tho

```suggestion
    if (isNorthstar || packageJson.private === true) {
      return false;
    }
```

Reply:
updated